### DISCUSSION TITLE
Windows Defender for 11

### DISCUSSION THREAD

**Discussion by CasualWolfGamer** [ionuttbara/windows-defender-remover#64] [Q&A]
Let's say I want to reinstall Defender, how would I do that? Do I need to wipe my pc entirely or something?

**Comment by ionuttbara** [comment]
upgrade in-place will re-add defender back.

**Reply by CasualWolfGamer** [reply]
how do i do that? I have windows 11 tho

**Reply by ionuttbara** [reply]
download from releases section , and run as a normal app.
